lib/rubocop/cop/rspec/indexed_let.rb

```diff
@@ -48,8 +48,8 @@ class IndexedLet < Base
         include AllowedIdentifiers
         include AllowedPattern
 
-        MSG = 'This `let` statement uses index in its name. Please give it ' \
-              'a meaningful name.'
+        MSG = 'This `let` statement uses `%<index>s` in its name. ' \
```

Comment:
Should we keep the work “index” eg “uses `3` index in its name”?

Reply:
`index` doesn't seem like the right word to use. I know the cop is called `IndexedLet` but I wonder if that is a misnomer, `NumberedLet` (or similar) seems more accurate to what it does.

Personally I would leave it as is but overall not that important to me if you want it changed. Let me know.